src/pybullet_blocks/envs/base_env.py

```diff
@@ -455,7 +455,7 @@ def step(
                     np.square(np.subtract(end_effector_position, block_position))
                 )
                 # Grasp successful.
-                if dist < 1e-6:
+                if dist < 1e-4:
```

Comment:
without this change, the grasping was sometimes flaky (even running the same code multiple times, unfortunately)